packages/react/src/components/createInlineOverlayComponent.tsx

```diff
@@ -132,14 +144,8 @@ export const createInlineOverlayComponent = <PropType, ElementType>(
             ? createElement(
                 'div',
                 {
-                  id: 'ion-react-wrapper',
                   ref: this.wrapperRef,
-                  className: 'ion-delegate-host',
-                  style: {
-                    display: 'flex',
-                    flexDirection: 'column',
-                    height: '100%',
```

Comment:
These styles were only needed for `ion-modal` and are now accomplished by the `.ion-page` class. See https://github.com/ionic-team/ionic-framework/pull/24510